src/fragments/lib/devpreview/ios/getting_started/30_preReq.mdx

```diff
@@ -0,0 +1,8 @@
+* [Install and configure Amplify CLI](https://docs.amplify.aws/cli/start/install)
+* An iOS application targeting at least iOS 13.0
```

Comment:
Should we add a minimum Xcode version (Swift version) here to be consistent with our existing pre-req section?